CHANGELOG.md

```diff
@@ -3,9 +3,11 @@
 All notable changes to the COBOL Language Support extension are documented in this file.
 
 ## [0.x.x] - TBD
-- Dependency file should contain 8 char max copybook name [#175](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/175)
-- StatusBar copybook downloader profile switcher [#239](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/239)
-- Move fetch copybook quickfix from LSP server to extension. [#235](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/235)
+- Redefine responsibilities of WorkspaceServiceTest [232](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/232) Thanks @[zacanbrcom](https://github.com/zacanbrcom)
+- Remove copybooks Quick Fix on the server side due to it is a client responsibility [234](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/234) Thanks @[temanbrcom](https://github.com/temanbrcom)
```

Comment:
This change message is in a little conflict with one line 10. I think we may remove this one